### ISSUE TITLE
404 gif not working

### ISSUE THREAD

**Issue by jamiehewitt15** [oceanprotocol/docs#1006]
**Describe the bug**
The gif on the 404 page is not working

**To Reproduce**
1. Go to a [page](https://docs.oceanprotocol.com/references/ocean_provider/vcvcvcvcv) that doesn't exist

**Screenshots**
![docs - 404 gif not working - 2022-05-27_16-25](https://user-images.githubusercontent.com/20739535/170730761-4e47486f-0cba-4c42-bc09-3635c11ad82f.png)

**Comment by akshay-ap** [comment]
I checked the API for fetching gif and it seems to work fine. We haven't done any changes that would impact gifs not rendering. 
I think checking of any dependency updates might have caused this side effect. Checking further.

**Comment by jamiehewitt15** [comment]
Is it necessary to have an API for fetching gifs? Maybe we can just have one gif for the 404 page, it would be better than this black rectangle

**Comment by akshay-ap** [comment]
> 

@kremalicious may be the right person to answer this. As far as I understand, random gifs were added just as a fun element for 404s.

**Comment by kremalicious** [comment]
So did you look into network tab of web inspector? What fails? If it's API key then a request would fail.

**Comment by jamiehewitt15** [comment]
Yeah good point. The giphy request is actually working fine, it's sending a 200 response with a gif
